pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Sink.java

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
I don't think we need to add it to v1 anymore, since admin cli has been switched to use v2

Reply:
deleted